src/Frontend/Core/Engine/Page.php

```diff
@@ -104,8 +104,14 @@ public function load()
         // set tracking cookie
         Model::getVisitorId();
 
-        // get pageId for requested URL
-        $this->pageId = Navigation::getPageId(implode('/', $this->URL->getPages()));
+        // get page content
+        $this->record = $this->getPageContent(
+            // get pageId for requested URL
```

Comment:
Blank lines and comments in method calls seem messy to me. Can't you put this in the comment above the method call?